google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/GenerateSignedPostPolicyV4.java

```diff
@@ -68,7 +68,7 @@ public static void generateSignedPostPolicyV4(
               + "' type='hidden' />\n");
     }
     htmlForm.append("  <input type='file' name='file'/><br />\n");
-    htmlForm.append("  <input type='submit' value='Upload File' name='submit'/><br />\n");
+    htmlForm.append("  <input type='submit' value='Upload File' name='x-ignore-submit'/><br />\n");
```

Comment:
In this case can you remove `name` attribute from submit?

Reply:
Yes, I believe so.